src/components/forms/ObjPaletteSelect.tsx

```diff
@@ -21,24 +22,30 @@ interface ObjPaletteOption {
   colors: string[];
 }
 
-const options: ObjPaletteOption[] = [
-  {
-    value: "OBP0",
-    label: "0: OBP0",
-    colors: ["E8F8E0", "B0F088", "", "202850"],
-  },
-  {
-    value: "OBP1",
-    label: "1: OBP1",
-    colors: ["E8F8E0", "509878", "", "202850"],
-  },
-];
-
 export const ObjPaletteSelect: FC<ObjPaletteSelectProps> = ({
   name,
   value = "OBP0",
   onChange,
 }) => {
+  const settings = useAppSelector((state) => state.project.present.settings);
+  const obp0 = settings.defaultOBP0;
+  const obp1 = settings.defaultOBP1;
+  const dmgPal = [settings.customColorsWhite, settings.customColorsLight, settings.customColorsDark, settings.customColorsBlack];
```

Comment:
I used the custom colors from settings instead of `DMG_PALETTE.colors` because I thought for sure I'd be making those configurable in this PR. I still plan on doing that, but I realized how cumbersome the task was when the various workers complained about trying to access settings off the main thread, so I abandoned ship there.